script.js

```diff
@@ -9,4 +9,224 @@ function closeMenu() {
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.',
+    descriptionDesktop: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.',
+    image: './BG/Snapshoot-Portfolio.png',
+    technologies: ['html', 'Ruby on rails', 'css'],
+    link: 'link_to_live_version.com',
+  },
+  {
+    name: 'Multi - Post stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.',
+    image: './BG/Snapshoot-Portfolio.png',
+    technologies: ['html', 'Ruby on rails', 'css'],
+    link: 'link_to_live_version.com',
+  },
+  {
+    name: 'Multi - Post stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.',
+    image: './BG/Snapshoot-Portfolio.png',
+    technologies: ['html', 'Ruby on rails', 'css'],
+    link: 'link_to_live_version.com',
+  },
+
+  {
+    name: 'Multi - Post stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.',
+    image: './BG/Snapshoot-Portfolio.png',
+    technologies: ['html', 'Ruby on rails', 'css'],
+    link: 'link_to_live_version.com',
+  },
+];
```

Comment:
- [ ] Great job making these changes correctly. But, I'm afraid you missed a hard requirement! Don't worry, you're almost there! 💪
Now that you've created the projects' array with js, don't you think it's better to add them dynamically to the DOM, because the array now includes everything required for a project to have, so the HTML hard-coding (as required earlier) for the projects' list should be added dynamically from the data restored in this array? Kindly notice that this is going to make your code less and the website optimized.

_Hint: The projects' list that already existed in the HTML should be removed and done dynamically with js._
_Later, adding projects in the future will be super easy. You will only add it in one place - as another object in the projects' array._
> Less code is better anywhere and anytime! 😄